LonnyGomes/urly
# Task #24 - koa unit tests

Fixes #24 

Depends on #23 (merge this first)

Adds koa unit tests and implements the API logic to match @panoz7's POC

## Verification test

- run `npm run build && npm start`
- navigate to `http://localhost:3000` and confirm functionality
- verify all unit tests pass
